ui/src/components/itemELement.js

```diff
@@ -14,6 +14,13 @@ class ItemElement extends React.Component{
             textClass="gridText"
             nameClass="gridName";
             priceClass="gridPrice";
+        }else if(this.props.type=="list"){
```

Comment:
use triple equals, almost always

review other places in code for this improvement also